src/main/java/com/epam/task/third/sorter/TriangleSorter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.task.third.sorter;
+
+import com.epam.task.third.comparator.TriangleAreaComparator;
+import com.epam.task.third.comparator.TriangleIdComparator;
+import com.epam.task.third.comparator.TrianglePerimeterComparator;
+import com.epam.task.third.figures.entities.Triangle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.epam.task.third.observer.TriangleObservable;
+
+import java.util.Collections;
+
+public class TriangleSorter {
```

Comment:
no need in this class and methods, just call  Collections.sort(triangleObservables, comparator); and pass specific comaprator (because this class is doing nothing)